### DISCUSSION TITLE
part-ii/2016-98/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#424] [Question comments]
Part II 2016 Number Theory: Paper 2, Section I, I 

https://questions.tripos.org/part-ii/2016-98/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

For the first calculation:

$$
\begin{align*}
(\frac{202}{11189}) &= (\frac{2}{11189})(\frac{101}{11189}) \\
&= -(\frac{101}{11189}) \\
&= -(\frac{11189}{101}) \\
&= -(\frac{79}{101}) \\
&= -(\frac{101}{79}) \\
&= -(\frac{22}{79}) \\
&= -(\frac{2}{79})(\frac{11}{79}) \\
&= -(\frac{11}{79}) \\
&= +(\frac{79}{11}) \\
&= +(\frac{2}{11}) \\
&= -1
\end{align*}
$$

For the second calculation:

$$
\begin{align*}
(\frac{974}{1001}) &= (\frac{2}{1001})(\frac{487}{1001}) \\
&= (\frac{487}{1001}) \\
&= (\frac{1001}{487}) \\
&= (\frac{27}{487}) \\
&= (\frac{3}{487}) \\
&= -(\frac{487}{3}) \\
&= -(\frac{1}{3}) \\
&= -1
\end{align*}
$$

</details>
